code/src/index.css

```diff
@@ -11,3 +11,242 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+/**************** LOADER *****************/
+
+.loader-page-container {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.loader-wrapper p {
+  color: white;
+}
+.popcorn {
+  width: 100px;
+}
+
+.loader-wrapper .popcorn:nth-last-child(1) {
+  animation: loader 0.6s 0.1s linear infinite;
+}
+.loader-wrapper .popcorn:nth-last-child(2) {
+  animation: loader 0.6s 0.2s linear infinite;
+}
+.loader-wrapper .popcorn:nth-last-child(3) {
+  animation: loader 0.6s 0.3s linear infinite;
+}
+
+@keyframes loader {
+  0 {
+    transform: translate(0, 0);
+  }
+  50% {
+    transform: translate(0, 25px);
+  }
+  100% {
+    transform: translate(0, 0);
+  }
+}
+
+/***************** HEADER ******************/
```

Comment:
Really nice and well structured CSS 🌟 . Good that you use a "top-to-bottom" approach, it makes it easier to follow and understand the thinking behind your styling. And good that you use comments, makes it easier to read 👏